### DISCUSSION TITLE
Discuss: Exploratory Annotation Interaction

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#52] [Ideas]
Discuss, or Question and answer about this project specifically.

**Comment by Apurv428** [comment]
Hi @birm , in this project description, it is mentioned to create a UX Workflow. So, do we need to develop or just create the user experience workflow using tools like figma?

**Reply by birm** [reply]
We would expect the tools to be functional and work with caMicroscope, but figma may be a good first step.

**Comment by psankhe28** [comment]
Hello @birm, I'm enthusiastic about the project and have a few questions:

1. Can you provide examples or concepts of what you define as a "novel and distinct functional annotation prototype" within the realm of medical imaging and caMicroscope?
2. Are there specific features or functionalities you would like to see in these prototypes?
3. What's the best way to contact the mentor for a detailed discussion on the implementation?
4. Additionally, are there any proof-of-concept tasks or preliminary work that should be completed before submitting a proposal?
